cli/azd/pkg/azdo/service_connection.go

```diff
@@ -170,18 +193,29 @@ func createAzureRMServiceEndPointArgs(
 		Scheme:     &endpointScheme,
 		Parameters: &endpointAuthorizationParameters,
 	}
+	description := "Azure Service Connection created by azd"
+
+	pRef := []serviceendpoint.ServiceEndpointProjectReference{{
+		Name:        &ServiceConnectionName,
+		Description: &description,
+		ProjectReference: &serviceendpoint.ProjectReference{
+			Id:   to.Ptr(uuid.MustParse(*projectId)),
```

Comment:
Should we validate this `projectId`? It will panic if it's not in the correct format.

Reply:
`projectId` is a value that azd reads after either creating a new project or selecting an axisting one.  Folks can't enter the id directly. So it should be totally a panic if the id becomes nil at any point